lib/domUpdates.js

```diff
@@ -28,21 +28,34 @@ const domUpdates = {
   displayPuzzleBoxes() {
     let answerArr = game.puzzle.splitAnswer();
     let row1 = $('.row1 div');
-    // console.log(row1)
     let row2 = $('.row2 div');
-    // console.log(row2)
     let row3 = $('.row3 div');
-    // console.log(row3)
     let row4 = $('.row4 div');
-    // console.log(row4)
-    console.log(answerArr[0]);
-    for (let i = 2; i <= 12; i++) {
-      if (answerArr[0].indexOf() !== ' ') {
-      $(row1[i]).css({'background': '#fff'});
 
-      }
-    }
-    // $(boxArray[2]).css({'background': '#fff'}).text('Y')
+    answerArr[0].forEach((item, index) => {
+      if (item === ' ') return;
+      $(row1[index+2]).addClass('tile').text(item).attr('data-value', item);
+    });
+    answerArr[1].forEach((item, index) => {
+      if (item === ' ') return;
+      $(row2[index+2]).addClass('tile').text(item).attr('data-value', item);
+    });
+    answerArr[2].forEach((item, index) => {
+      if (item === ' ') return;
+      $(row3[index+2]).addClass('tile').text(item).attr('data-value', item);
+    });
+    answerArr[3].forEach((item, index) => {
+      if (item === ' ') return;
+      $(row4[index+2]).addClass('tile').text(item).attr('data-value', item);
+    });
+  },
+
+  showLetters(letter) {
+    const letters = $(`[data-value=${letter}]`);
+    const letterSum = letters.length;
+    // access current spin element and multiply by letterSum to add to round points if correct
+    // Update player's points
+    letters.removeClass('tile').css({'color': '#000', 'font-size': '2em', 'background': '#fff'});
   },
```

Comment:
oh wait i see you refactored so it adds a class, im assuming we need them to get rid of them for each round?